src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -55,8 +55,11 @@ public class EditCommand extends Command {
             + PREFIX_PHONE + "91234567 "
             + PREFIX_EMAIL + "johndoe@example.com";
 
-    public static final String MESSAGE_EDIT_HELP = "Edit Command: (Coming soon)\n"
-            + "Format: (Coming soon)\n";
+    public static final String MESSAGE_EDIT_HELP = "Edit Command: Edits a client's particulars through their "
+            + "index number.\n"
+            + "Format: edit INDEX NUMBER [n/NAME] [p/PHONE] [i/INCOME] [e/EMAIL] [a/ADDRESS] [f/FAMILY] "
```

Comment:
this is a nice touch because it suggests that when we edit, we can edit specific fields only (since it is in the square brackets so it is optional)